src/datadog/azext_datadog/generated/_help.py

```diff
@@ -0,0 +1,343 @@
+linking-auth-code="someAuthCode" linking-client-id="00000000-0000-0000-0000-000000000000" subscription="pro" \
+--monitoring-status "Enabled" --user-info name="Alice" email-address="alice@microsoft.com" phone-number="123-456-7890" \
+--sku-name "drawdown_testing_20200904_Monthly" --tags Environment="Dev" --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor update'] = """
+    type: command
+    short-summary: "Update a monitor resource."
+    examples:
+      - name: Monitors_Update
+        text: |-
+               az datadog monitor update --name "myMonitor" --monitoring-status "Enabled" --tags Environment="Dev" \
+--resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor delete'] = """
+    type: command
+    short-summary: "Delete a monitor resource."
+    examples:
+      - name: Monitors_Delete
+        text: |-
+               az datadog monitor delete --name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the datadog monitor is met.
+    examples:
+      - name: Pause executing next line of CLI script until the datadog monitor is successfully created.
+        text: |-
+               az datadog monitor wait --name "myMonitor" --resource-group "myResourceGroup" --created
+      - name: Pause executing next line of CLI script until the datadog monitor is successfully deleted.
+        text: |-
+               az datadog monitor wait --name "myMonitor" --resource-group "myResourceGroup" --deleted
+"""
+
+helps['datadog refresh-set-password'] = """
+    type: group
+    short-summary: datadog refresh-set-password
+"""
+
+helps['datadog refresh-set-password get'] = """
+    type: command
+    short-summary: "Refresh the set password link and return a latest one."
+    examples:
+      - name: RefreshSetPassword_Get
+        text: |-
+               az datadog refresh-set-password get --monitor-name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog tag-rule'] = """
+    type: group
+    short-summary: datadog tag-rule
+"""
+
+helps['datadog tag-rule list'] = """
+    type: command
+    short-summary: "List the tag rules for a given monitor resource."
+    examples:
+      - name: TagRules_List
+        text: |-
+               az datadog tag-rule list --monitor-name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog tag-rule show'] = """
+    type: command
+    short-summary: "Get a tag rule set for a given monitor resource."
+    examples:
+      - name: TagRules_Get
+        text: |-
+               az datadog tag-rule show --monitor-name "myMonitor" --resource-group "myResourceGroup" --rule-set-name \
+"default"
+"""
+
+helps['datadog tag-rule create'] = """
+    type: command
+    short-summary: "Create or update a tag rule set for a given monitor resource."
```

Comment:
The short summary needs to remove `or update` as this is for the `create` command.

Reply:
Updated.